decompiler/extractor/main.cpp

```diff
@@ -407,6 +432,19 @@ int main(int argc, char** argv) {
       }
       path_to_iso_files = data_dir_path;
     }
+
+    // write out a json file with some metadata for the game
+    nlohmann::json buildinfo_json;
+    auto flags_json = nlohmann::json::array();
+    for (const auto& [n, f] : sGameIsoFlagNames) {
+      if (flags & f) {
+        flags_json.push_back(n);
+      }
+    }
+    buildinfo_json["flags"] = flags_json;
+    // something tells me a ps2 game is unlikely to have a json file in root
+    file_util::write_text_file((path_to_iso_files / "buildinfo.json").string(),
+                               buildinfo_json.dump(2));
```

Comment:
looks like we are only writing out the flags to this file?  Is this just to figure out what game was extracted?  If so why not just write out the serial/ELF_HASH (the keys to retrieve the same release info from the db).

Reply:
I originally was gonna write the whole iso metadata here but ended up just doing flags. I didn't want to just have the elf hash in case the elf is identical but the other files in the disc aren't. I'm not sure this is the case (haven't checked yet) so I went with the safer option.